package.json

```diff
@@ -7,10 +7,9 @@
     "test": "npm run test:typecheck && npm run test:lint && npm run test:format && npm run test:rust",
     "test:typecheck": "tsc",
     "test:rust": "cargo test",
-    "test:lint": "npm run test:lint:rust && npm run test:lint:ts",
+    "test:lint": "npm run test:lint:ts",
     "test:lint:ts": "biome lint",
-    "test:lint:rust": "cargo clippy --all --no-deps --all-targets --all-features -- -D warnings",
-    "test:format": "npm run test:format:rust && npm run test:format:ts",
+    "test:format": "npm run test:format:ts",
```

Comment:
I find the split between tasks in mise toml files and package.json to be a bit confusing. What's the boundary between the two? It looks like there are both npm and cargo commands in both. I reckon we either go with one or the other.

Reply:
The plan is to go all in on mise as we have elsewhere but I didn't want to do more than was necessary in this PR.